contracts/Utils.sol

```diff
@@ -0,0 +1,34 @@
+pragma solidity ^0.5.11;
+
+library Utils {
+    function uint2str(uint256 _i)
+        internal
+        pure
+        returns (string memory _uintAsString)
+    {
+        if (_i == 0) {
+            return '0';
+        }
+        uint256 j = _i;
+        uint256 len;
+        while (j != 0) {
+            len++;
+            j /= 10;
+        }
+        bytes memory bstr = new bytes(len);
+        uint256 k = len - 1;
+        while (_i != 0) {
+            bstr[k--] = byte(uint8(48 + (_i % 10)));
+            _i /= 10;
+        }
+        return string(bstr);
+    }
+
+    function toPayable(address _a)
+        internal
+        pure
+        returns (address payable _addressAsPayable)
+    {
+        return address(uint160(_a));
```

Comment:
Do you need a function just for this one liner?

Reply:
Not really, has been removed with this [commit](https://github.com/wearekickback/contracts/pull/76/commits/627316748da876795f5f16114ec2ae38111c5544).